scripts/generate_contributors.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+/*
+ * This script fetches a maximum of PAGE_COUNT * 30 contributors and writes
+ * their usernames and url to the Contributors.md file.
+ */
+
+const fetch = require('node-fetch');
+const fs = require('fs');
+
+const PAGE_COUNT = 4;
+const FILE_NAME = "Contributors.md";
+const BASE_URL = "https://api.github.com/repos/sb2nov/mac-setup/contributors?page=";
+const HEADER = `# Contributors
+
+Thank you everyone that have contributed to creating this awesome guide. Here are the names of a few; for the full list please visit the [GitHub Contributor page](https://github.com/sb2nov/mac-setup/graphs/contributors).
+
+`;
+
+const fetchContributor = async url => {
+  try {
+    const response = await fetch(url);
+    const json = await response.json();
+    return json;
+  } catch (err) {
+    console.error(err);
```

Comment:
the problem with silently discarding this function is that the `Promise` on line 30 will fail. Would also prefer some sort of validation on the contributor name (prevents xss injections into the website)

Reply:
How can we make this fail, by throwing?

I really hope GitHub does XSS injection validation on usernames 😄 But sure, what exactly did you have in mind? Is checking for a `<script>` tag enough?

Reply:
I do hope so too :) yes, checking for a script tag would be a start. I will see if there are any more validations to be added later. for a start this should be good enough.

As for the silent gobbling of the exception, yes a throw would be better IMO

Reply:
Fixed!